solc/CommandLineParser.cpp

```diff
@@ -909,6 +920,25 @@ General Information)").c_str(),
 	if (m_args.count(g_strBasePath))
 		m_options.input.basePath = m_args[g_strBasePath].as<string>();
 
+	if (m_args.count(g_strIncludePath) > 0)
+	{
+		if (m_options.input.basePath.empty())
+		{
+			serr() << "--" << g_strIncludePath << " option requires a non-empty base path." << endl;
+			return false;
+		}
+
+		for (string const& includePath: m_args[g_strIncludePath].as<vector<string>>())
+		{
+			if (includePath.empty())
+			{
+				serr() << "Empty values are not allowed in --" << g_strIncludePath << "." << endl;
+				return false;
+			}
+			m_options.input.includePaths.push_back(includePath);
```

Comment:
Could `includePath` be moved here?

Reply:
Probably not. I think that `.as()` returns a reference to a string stored by `program_options`. But I need to check to say for sure.

Reply:
ok!

Reply:
It does return a reference so I left it as is.